Adding first US-RSE history post! (#686)
USRSE/usrse.github.io: _posts/history/2022-02-06-a-brief-history-of-usrse.md

The [grant](https://sloan.org/grant-detail/9227) was funded and a workshop was scheduled for April 2020 in Princeton, NJ.
Unfortunately, it was bad timing, and COVID-19 forced the repeated postponement of the workshop; as of Feb 2022, it is planned for April 2022, COVID-permitting.

### Community Voices

In July of 2019, after joining US-RSE in May, community member Vanessa Sochat was frustrated about the message being sent about what constituted an RSE.
Her first effort to send a message of a diversity of roles was to make a video, "The Story of the Research Software Engineer" [[ref](https://www.youtube.com/watch?v=trAfA9VWLTQ&t=3s)], however any video can only have limited impact.
She woke up one morning in September, and knew that she needed to start a podcast, and lift up research software engineers from all around the community to tell their stories directly.
This became the RSE Stories podcast [[ref](https://us-rse.org/rse-stories)] which eventually brought on a host from the UK, and inspired the creation of other RSE-run podcasts [[ref](https://codeforthought.buzzsprout.com/)].

### Community Calls and Newsletters

> US-RSE started creating and distributing [newsletters](https://us-rse.org/newsletters/) in July 2019, led by Sandra Gesing, and has generally tried to issue them every two months, as one of our means of sharing information among members. They began as an experiment in moderated online discussion with two broad goals:

Reviewer: ```suggestion
Community calls began as an experiment in moderated online discussion with two broad goals:
```

* sharing ideas, knowledge and practices, and also 
* giving everyone a chance to talk to each other, listen to each other and learn about and from one another. 

Over time they evolved to a base one hour format with focussed discussion on some topical theme (technical or organizational) and usually mixed a short presentation with small group discussions around topical prompts.
Small group discussions are kept deliberately small (3-4 people) to ensure that conversations are not overly monopolized by single voices and a broad spectrum of experience and knowledge is mutually shared.
Brief plenary reviews after group discussions provide a way for all participants to compare conversations and communicate collectively.
The community call notes archive and video recordings provide a growing raw resource for RSE learning and knowledge sharing [[Google drive folder](https://drive.google.com/drive/u/0/folders/1hzKqwPUs6IL42sJ8jV8ZjA_YSeXF6-qH)] from practicing RSEs across disciplines, experience levels and economic sectors.
They cover all manner of topics, spanning developing professional career profiles to tools for code review.

> 

Reviewer: ```suggestion

US-RSE started creating and distributing [newsletters](https://us-rse.org/newsletters/) in July 2019, led by Sandra Gesing, and has generally tried to issue them every two months, as one of our means of sharing information among members. 

```

### Leadership

Ian Cosden continued to take the lead in pushing US-RSE forward to a more formal organization.
An email from May 14, 2019 established a steering committee for US-RSE:

> Dear all [Jay Jay Billings, Jeff Carver, Charles Ferenbaugh, Sandra Gesing, Chris Hill, Dan Katz, Christina Maimone, Lance Parsons, Jordan Perr-Sauer],
>
> In the last couple months the US-RSE group has clearly seen a rise in interest, membership, and engagement.
> The growth is extremely encouraging, and with a small amount of effort, it’s only a matter of time before we reach a significant level of activity. 
>
> I wanted to reach out, not via slack, but in a more composed email.
> The nine of you (plus me) are all listed on the website as the current steering committee, for the sole reason that you either (a) wrote your name as a contributor in the document that helped define our mission or (b) expressed interest in participating around the same time.
> When I put the new website together, I just ran with it and appointed us the "Steering Committee."

> For better or worse, that’s all it took in December. 

Reviewer: ```suggestion
> For better or worse, that’s all it took in December. 
```

> 

Reviewer: ```suggestion
>
```

Reviewer: to keep the quote as a single quote, rather than the two it displays as now

Author: ah thanks! Looks like I just forgot one :)

![PEARC19 BOF]({{ site.baseurl }}/assets/posts/history/2022-02-brief-history-usrse/image3.png "The PEARC19 BOF (image credit: Dan Katz)")

The PEARC19 BOF (Birds of a Feather), "Building a Community of Research Software Engineers" was the first large event/conference to announce the existence of US-RSE.
In preparation for PEARC19 and future conferences, Ian Cosden had 2,000 stickers printed with the US-RSE logo (see picture below).
Always the optimist, he thought it necessary to bring the entire box of stickers to PEARC19.
It almost didn’t make it, because in the Philadelphia airport his carryon was flagged as highly suspicious.
It seems that a box of stickers looks very similar to explosives.
After a surprisingly thorough search the TSA agent laughed and let the stickers make the trip to Chicago.
Unfortunately for Ian, he was only able to give out a handful of stickers, which meant he had to return with a nearly full box.
He was stopped and searched again in the Chicago airport on the way home. 

<div style="padding:20px; width:600px">
<img src="{{ site.baseurl }}/assets/posts/history/2022-02-brief-history-usrse/image5.png"/>
</div>

> Another major venue was SC19:

Reviewer: ```suggestion
Another major venue was SC19, where a workshop was held, and a number of workshop attendee, both US and international RSEs met socially in the evening:
```

The chart shows change over time, while the map shows a single snapshot in time of membership based on those who choose to report their city and state.
In a larger sense, what these small initiatives capture is the general excitement that was held by the community - not just to find others like themselves, but to feel like they can be a part of a larger community and movement.
And because many community members are software engineers, it was fairly natural to have things like automation, continuous integration, and new web interfaces added to make this possible.

In mid-2019, Ian Cosden, Jeff Carver, and Dan Katz, in an effort to seek funding to grow the organization, began conversations with The Alfred P. Sloan Foundation.
The three wrote a proposal for a dedicated, in-person, [US-RSE community building workshop](https://us-rse.org/first-community-workshop/) designed to bring together community members to plan the best path forward for US-RSE.
The [grant](https://sloan.org/grant-detail/9227) was funded and a workshop was scheduled for April 2020 in Princeton, NJ.

> Unfortunately, it was bad timing, and COVID-19 forced the repeated postponement of the workshop. 

Reviewer: ```suggestion
Unfortunately, it was bad timing, and COVID-19 forced the repeated postponement of the workshop; as of Feb 2022, it is planned for April 2022, COVID-permitting.
```

### Community Voices

In July of 2019, after joining US-RSE in May, community member Vanessa Sochat was frustrated about the message being sent about what constituted an RSE.
Her first effort to send a message of a diversity of roles was to make a video, "The Story of the Research Software Engineer" [[ref](https://www.youtube.com/watch?v=trAfA9VWLTQ&t=3s)], however any video can only have limited impact.
She woke up one morning in September, and knew that she needed to start a podcast, and lift up research software engineers from all around the community to tell their stories directly.
This became the RSE Stories podcast [[ref](https://us-rse.org/rse-stories)] which eventually brought on a host from the UK, and inspired the creation of other RSE-run podcasts [[ref](https://codeforthought.buzzsprout.com/)].

### Community Events

> TBD: workshops, conference events, etc. that were organized – definitely the two actual US-RSE events (don't remember what they were called) – virtual mini-conferences

Reviewer: We shouldn't publish this with a TBD...

Author: Just an oversight - removed!

### Community Calls and Newsletters

> TBD - should be put in as a major community effort - first community call and first newsletter were both in July 2019

Reviewer: another TBD that needs to be fleshed out

Author: We can also just remove the section.

Reviewer: we could remove this sentence, since the community calls are discussed here - but maybe we should add something about the newsletters?

Reviewer: Maybe "US-RSE started creating and distributing [newsletters](https://us-rse.org/newsletters/) in July 2019, led by Sandra Gesing, and has generally tried to issue them every two months, as one of our means of sharing information among members."

Author: Done!

Community calls to connect across the US-RSE network were started in July 2019. They began as an experiment in moderated online discussion with two broad goals 

* sharing ideas, knowledge and practices, and also 
* giving everyone a chance to talk to each other, listen to each other and learn about and from one another. 

Over time they evolved to a base one hour format with focussed discussion on some topical theme (technical or organizational) and usually mixed a short presentation with small group discussions around topical prompts.
Small group discussions are kept deliberately small (3-4 people) to ensure that conversations are not overly monopolized by single voices and a broad spectrum of experience and knowledge is mutually shared.
Brief plenary reviews after group discussions provide a way for all participants to compare conversations and communicate collectively.

> The community call notes archive and video recordings provide a growing raw resource for RSE learning and knowledge sharing [[https://drive.google.com/drive/u/0/folders/1hzKqwPUs6IL42sJ8jV8ZjA_YSeXF6-qH](https://drive.google.com/drive/u/0/folders/1hzKqwPUs6IL42sJ8jV8ZjA_YSeXF6-qH)] from practicing RSEs across disciplines, experience levels and economic sectors.

Reviewer: ```suggestion
The community call notes archive and video recordings provide a growing raw resource for RSE learning and knowledge sharing [[Google drive folder](https://drive.google.com/drive/u/0/folders/1hzKqwPUs6IL42sJ8jV8ZjA_YSeXF6-qH)] from practicing RSEs across disciplines, experience levels and economic sectors.
```

They cover all manner of topics, spanning developing professional career profiles to tools for code review.

### Working Groups

> The US-RSE [governance document](https://us-rse.org/about/governance/)established a need and role for working groups, but these were informal at first.

Reviewer: ```suggestion
The US-RSE [governance document](https://us-rse.org/about/governance/) established a need and role for working groups, but these were informal at first.
```

> Note that we don't say anything about diversity in [https://us-rse.org](https://us-rse.org) or [https://us-rse.org/about/](https://us-rse.org/about/) or [https://us-rse.org/mission/](https://us-rse.org/mission/)  I think we probably should have increasing diversity of the US-RSE community (not our just members, but the larger community) as a goal.

While DEI wasn't a new idea as US-RSE had talked about diversity from an early point, it was a gap in our community to not explicitly do anything about it. These early discussions led to diversity, equity, and inclusion being the topic of our [October 2020 community call](https://us-rse.org/2020-10-13-community-call-dei/), and this in turn led to the DEI slack channel and the DEI working group led by Lance Parsons. 
The working group put together a DEI mission statement, with contributions from the entire group and leadership on the part of Alex Koufos, and an addition of DEI to the US-RSE Mission Statement.

In January 2021, community members Sajid Ali took the lead to establish a DEI Speaker Series [[ref](https://docs.google.com/document/d/1ezlG5ys40LMIlv83OtFfbviHpTY5EgFM86m6vVJkTPE/edit#)], which runs quarterly.

> In May, Nicole Brewer initiated the DEI book club club [[ref](https://docs.google.com/document/d/1GMMwjdq97_AFjAuDhaJ1FtlTc3weHerH6jHaB04bN3w/edit)] that started with a reading of <span style="text-decoration:underline;">Better Allies: Everyday Actions to Create Inclusive, Engaging Workplaces” by Karen Catlin</span> which was later expanded by Miranda Mundt to include all forms of media, such as podcasts.

Reviewer: ```suggestion
In May, Nicole Brewer initiated the DEI book club club [[ref](https://docs.google.com/document/d/1GMMwjdq97_AFjAuDhaJ1FtlTc3weHerH6jHaB04bN3w/edit)] that started with a reading of <span style="text-decoration:underline;">Better Allies: Everyday Actions to Create Inclusive, Engaging Workplaces” by Karen Catlin</span>, and which was later expanded by Miranda Mundt to include all forms of media, such as podcasts.
```

Author: This reads weird to me, but the original sentence was off too. What about a period, and then "The book club was later expanded..." and get rid of the which?

Reviewer: ok

### Financial Status

In November of 2020, community member Vanessa Sochat won a Science Slam, and wanted to contribute her prize to US-RSE.
When it was apparent that there was no means to do this, the steering committee put their heads together to figure out how to create a financial entity as part of US-RSE to not only be able to use the contribution, but to allow regular contributions from other community members. 
Dan and Chris investigated a set of options, and proposed to the steering committee in March 2021 that US-RSE establish a US-RSE Open Collective [[ref](https://opencollective.com/usrse)] and create a Treasurer role, which was proposed to the full US-RSE group in April 2021 [[ref](https://www.google.com/url?q=https://docs.google.com/document/d/15Se_gZ6L9lwk6zFMi5NbR8A4O9jdroyVZKCdWGrJxbU/edit&sa=D&source=docs&ust=1641157652243694&usg=AOvVaw2LHW4nO21xsSa2xBOZNhlb)], without any dissension.
After this, Chris Hill was elected to be the first treasurer, and our financial information was added to the website [[ref](https://us-rse.org/about/finances/)].
Through the Open Collective umbrella US-RSE was established as a non-profit 501c3 activity with a public service focus.

> A first set of financial goals for the organization was discussed at the 2021 Annual General Meeting [[https://docs.google.com/presentation/d/19-yTp2Q4SWP5ZiYb3VJa82hflc0p5FkeuhKWlIbwsD4](https://docs.google.com/presentation/d/19-yTp2Q4SWP5ZiYb3VJa82hflc0p5FkeuhKWlIbwsD4)].

Reviewer: ```suggestion
A first set of financial goals for the organization was discussed at the 2021 Annual General Meeting [[ref](https://docs.google.com/presentation/d/19-yTp2Q4SWP5ZiYb3VJa82hflc0p5FkeuhKWlIbwsD4)].
```

Initial guidelines were drafted and shared with the community for input and discussion in July 2019.
The Code of Conduct was established in October 2019, and the US-RSE [governance document](https://us-rse.org/about/governance/) was approved by the Steering Committee in June 2020.
The governance document established a process for elections for the Steering Committee, and, after Elections Chair Christina Maimone established election logistics, the first elections took place in December 2020.
Each election so far has brought a new member to the Steering Committee.
It has since evolved into a democratic process that we hope to further improve upon.
In particular, we recognize that our election process can be improved to further give opportunity to community members that have not yet had influence.

### Annual General Meetings

An annual meeting was an early goal for US-RSE.
While the initial hope was for an in-person event or conference, the pandemic put planning for such gatherings on hold.
Yet as the organization grew and processes for elections and finances were established, the need for some type of membership meeting became clear.

> The first Annual General Meeting was held in [2020](https://youtu.be/mcl3QY1lBzY) and continued in [2021](https://youtu.be/iKIbS16D8XU).

Reviewer: ```suggestion
The first Annual General Meeting was held in [2020](https://youtu.be/mcl3QY1lBzY), with a second in [2021](https://youtu.be/iKIbS16D8XU).
```

> It has become an opportunity to reflect on the yearly progress of the US-RSE, address the business of the organization, and bring together the general membership for a discussion.  

Reviewer: ```suggestion
These have become an opportunity to reflect on the yearly progress of the US-RSE, address the business of the organization including hearing from Steering Committee candidates, and bring together the general membership for a discussion.  
```
